static/app/components/feedback/feedbackSearch.tsx

```diff
@@ -35,6 +35,7 @@ const EXCLUDED_TAGS: string[] = [
   'os',
   'user',
   FieldKey.PLATFORM,
+  'feedback.num_feedbacks_used',
```

Comment:
idk if we should special case this if it wasn't shipped into any customer orgs or public sdks. 
sounds like this was temporarily part of some testing, and is added to some replays inside the our org, but no others?